homeassistant/components/tado/climate.py

```diff
@@ -224,21 +231,21 @@ class TadoClimate(TadoZoneEntity, ClimateEntity):
 
     def __init__(
         self,
-        tado,
-        zone_name,
-        zone_id,
-        zone_type,
-        heat_min_temp,
-        heat_max_temp,
-        heat_step,
-        cool_min_temp,
-        cool_max_temp,
-        cool_step,
-        supported_hvac_modes,
-        supported_fan_modes,
-        support_flags,
-        device_info,
-    ):
+        tado: TadoConnector,
+        zone_name: str,
+        zone_id: int,
+        zone_type: str,
+        supported_hvac_modes: list[HVACMode],
+        support_flags: ClimateEntityFeature,
+        device_info: dict[str, str],
+        heat_min_temp: Optional[float] = None,
```

Comment:
```suggestion
        heat_min_temp: float | None = None,
```

Please adopt the other places too